src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -107,7 +107,9 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                             var optionsMonitor = languageServer.Services.GetRequiredService<RazorLSPOptionsMonitor>();
 
                             // Explicitly not passing in the same CancellationToken as that might get cancelled before the update happens.
+#pragma warning disable CA2016 // Forward the 'CancellationToken' parameter to methods
```

Comment:
Why not pass through the token?

Reply:
> Explicitly not passing in the same CancellationToken as that might get cancelled before the update happens.

Reply:
This is on initialized though, if it gets cancelled the whole server gets torn down

Reply:
Ah not sure then. I just added the pragma based on the documentation above. I can remove it in that case so we can diagnose accordingly.